vendor/github.com/grafana/regexp/README.md

```diff
@@ -0,0 +1,12 @@
+# Grafana Go regexp package
```

Comment:
I'm surprised to see that this wasn't a dependency before. This is a relevant change.

Reply:
I think we should migrate Mimir to `grafana/regexp` too. I will do it in a separate PR.